From 48d352f637b71e60f13f118f1032fbfd5d707dfb Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Mon, 26 Aug 2019 10:26:59 +0200
Subject: [PATCH 1/7] test/helper: disable simplecov and minitest plugins

---
 test/helper.rb | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/test/helper.rb b/test/helper.rb
index 479cdf9c..1d53f147 100644
--- a/test/helper.rb
+++ b/test/helper.rb
@@ -11,24 +11,10 @@ def jruby?
   defined?(RUBY_ENGINE) && RUBY_ENGINE == "jruby"
 end
 
-if ENV["CI"]
-  require "simplecov"
-  SimpleCov.start
-else
-  require File.expand_path("simplecov_custom_profile", __dir__)
-  SimpleCov.start "gem" do
-    add_filter "/vendor/gem"
-    add_filter "/vendor/bundle"
-    add_filter ".bundle"
-  end
-end
-
 require "nokogiri"
 require "rubygems"
 require "ostruct"
 require "minitest/autorun"
-require "minitest/reporters"
-require "minitest/profile"
 require "rspec/mocks"
 require_relative "../lib/jekyll"
 
@@ -41,13 +27,6 @@ include Jekyll
 
 require "jekyll/commands/serve/servlet"
 
-# Report with color.
-Minitest::Reporters.use! [
-  Minitest::Reporters::DefaultReporter.new(
-    :color => true
-  ),
-]
-
 module Minitest::Assertions
   def assert_exist(filename, msg = nil)
     msg = message(msg) { "Expected '#{filename}' to exist" }
-- 
2.26.2

