From bc65b740dbdb2d48775c60d5f0341cf0c3e58501 Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Mon, 26 Aug 2019 10:27:48 +0200
Subject: [PATCH] test/utils: remove internet connectivity test


diff --git a/test/test_utils.rb b/test/test_utils.rb
index f282cb38a..09810ea06 100644
--- a/test/test_utils.rb
+++ b/test/test_utils.rb
@@ -429,10 +429,4 @@ class TestUtils < JekyllUnitTest
       assert_equal "bom|another", merged[:encoding]
     end
   end
-
-  context "Utils::Internet.connected?" do
-    should "return true if there's internet" do
-      assert Utils::Internet.connected?
-    end
-  end
 end
-- 
2.48.1

