From 3740b9ca0b174ffa1f671015315f20730950b00e Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Mon, 26 Aug 2019 10:27:48 +0200
Subject: [PATCH 2/8] test/utils: remove internet connectivity test

---
 test/test_utils.rb | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/test/test_utils.rb b/test/test_utils.rb
index 7f230c378..0918f1fa2 100644
--- a/test/test_utils.rb
+++ b/test/test_utils.rb
@@ -429,10 +429,4 @@ class TestUtils < JekyllUnitTest
       assert_equal "bom|another", merged[:encoding]
     end
   end
-
-  context "Utils::Internet.connected?" do
-    should "return true if there's internet" do
-      assert Utils::Internet.connected?
-    end
-  end
 end
-- 
2.39.1

