From 42bca9bb19505b8b3164b0473cd2e004adb8d8a9 Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Wed, 19 Sep 2018 12:18:21 +0200
Subject: [PATCH 03/10] test/utils: remove internet connectivity test

---
 test/test_utils.rb | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/test/test_utils.rb b/test/test_utils.rb
index 844ef825..596bd8f9 100644
--- a/test/test_utils.rb
+++ b/test/test_utils.rb
@@ -415,9 +415,4 @@ class TestUtils < JekyllUnitTest
     end
   end
 
-  context "Utils::Internet.connected?" do
-    should "return true if there's internet" do
-      assert Utils::Internet.connected?
-    end
-  end
 end
-- 
2.21.0

