From 1f1dbb8e5bfca51d9eb086bd378ec1453f0f88f5 Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Wed, 19 Sep 2018 12:22:32 +0200
Subject: [PATCH 5/9] tests/configuration: disable tests requiring the tomlrb
 gem

---
 test/test_configuration.rb | 33 ---------------------------------
 1 file changed, 33 deletions(-)

diff --git a/test/test_configuration.rb b/test/test_configuration.rb
index 17c0d219..f4434639 100644
--- a/test/test_configuration.rb
+++ b/test/test_configuration.rb
@@ -367,39 +367,6 @@ class TestConfiguration < JekyllUnitTest
         Jekyll.configuration(test_config.merge({ "config" => [@paths[:empty]] }))
     end
 
-    should "successfully load a TOML file" do
-      Jekyll.logger.log_level = :warn
-      assert_equal \
-        site_configuration({
-          "baseurl" => "/you-beautiful-blog-you",
-          "title"   => "My magnificent site, wut",
-          "config"  => [@paths[:toml]],
-        }),
-        Jekyll.configuration(test_config.merge({ "config" => [@paths[:toml]] }))
-      Jekyll.logger.log_level = :info
-    end
-
-    should "load multiple config files" do
-      External.require_with_graceful_fail("tomlrb")
-
-      allow(SafeYAML).to receive(:load_file).with(@paths[:default]).and_return({})
-      allow(SafeYAML).to receive(:load_file).with(@paths[:other]).and_return({})
-      allow(Tomlrb).to receive(:load_file).with(@paths[:toml]).and_return({})
-      allow($stdout).to receive(:puts).with("Configuration file: #{@paths[:default]}")
-      allow($stdout).to receive(:puts).with("Configuration file: #{@paths[:other]}")
-      allow($stdout).to receive(:puts).with("Configuration file: #{@paths[:toml]}")
-      assert_equal(
-        site_configuration({
-          "config" => [@paths[:default], @paths[:other], @paths[:toml]],
-        }),
-        Jekyll.configuration(
-          test_config.merge(
-            { "config" => [@paths[:default], @paths[:other], @paths[:toml]] }
-          )
-        )
-      )
-    end
-
     should "load multiple config files and last config should win" do
       allow(SafeYAML)
         .to receive(:load_file)
-- 
2.19.1

