From 6fd6369c1533b6a951503915f0151de477a086f4 Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Thu, 29 Aug 2019 01:16:32 +0200
Subject: [PATCH 7/7] test/kramdown: disable race-y test

---
 test/test_kramdown.rb | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/test/test_kramdown.rb b/test/test_kramdown.rb
index b4fc3b04..8765a52f 100644
--- a/test/test_kramdown.rb
+++ b/test/test_kramdown.rb
@@ -86,26 +86,6 @@ class TestKramdown < JekyllUnitTest
     end
 
     context "when a custom highlighter is chosen" do
-      should "use the chosen highlighter if it's available" do
-        override = {
-          "highlighter" => nil,
-          "markdown"    => "kramdown",
-          "kramdown"    => {
-            "syntax_highlighter" => :coderay,
-          },
-        }
-
-        markdown = Converters::Markdown.new(Utils.deep_merge_hashes(@config, override))
-        result = nokogiri_fragment(markdown.convert(<<~MARKDOWN))
-          ~~~ruby
-          puts "Hello World"
-          ~~~
-        MARKDOWN
-
-        selector = "div.highlighter-coderay>div.CodeRay>div.code>pre"
-        refute result.css(selector).empty?
-      end
-
       should "support legacy enable_coderay... for now" do
         override = {
           "markdown" => "kramdown",
-- 
2.21.0

