From 829afea65e38bca3f8fc94db3f323e58d683afe2 Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Thu, 29 Aug 2019 01:16:32 +0200
Subject: [PATCH 7/7] test/kramdown: disable race-y test

---
 test/test_kramdown.rb | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/test/test_kramdown.rb b/test/test_kramdown.rb
index 437e2a1c..0ce9bb1b 100644
--- a/test/test_kramdown.rb
+++ b/test/test_kramdown.rb
@@ -98,24 +98,6 @@ class TestKramdown < JekyllUnitTest
     end
 
     context "when a custom highlighter is chosen" do
-      should "use the chosen highlighter if it's available" do
-        override = {
-          "highlighter" => nil,
-          "kramdown"    => {
-            "syntax_highlighter" => "coderay",
-          },
-        }
-        converter = fixture_converter(Utils.deep_merge_hashes(@config, override))
-        result = nokogiri_fragment(converter.convert(<<~MARKDOWN))
-          ~~~ruby
-          puts "Hello World"
-          ~~~
-        MARKDOWN
-
-        selector = "div.highlighter-coderay>div.CodeRay>div.code>pre"
-        refute result.css(selector).empty?
-      end
-
       should "support legacy enable_coderay... for now" do
         override = {
           "kramdown" => {
-- 
2.26.2

