From fb5ae8a8ef8e0cb30236463de1b7e3a832a5d6f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADt=20Ondruch?= <vondruch@redhat.com>
Date: Mon, 4 Sep 2023 17:55:16 +0200
Subject: [PATCH] Test suite uses shoulda-context only.

---
 test/helper.rb | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/test/helper.rb b/test/helper.rb
index 3ec9d75cbdf..37258e46b60 100644
--- a/test/helper.rb
+++ b/test/helper.rb
@@ -35,7 +35,7 @@ def jruby?
 Jekyll.logger = Logger.new(StringIO.new, :error)
 
 require "kramdown"
-require "shoulda"
+require "shoulda-context"
 
 include Jekyll
 
