diff -urp jekyll-4.3.4.orig/test/test_filters.rb jekyll-4.3.4/test/test_filters.rb
--- jekyll-4.3.4.orig/test/test_filters.rb	2024-11-29 15:06:55.451227581 +0900
+++ jekyll-4.3.4/test/test_filters.rb	2024-11-29 15:12:38.161125064 +0900
@@ -1445,7 +1445,7 @@ class TestFilters < JekyllUnitTest
 
     context "inspect filter" do
       should "return a HTML-escaped string representation of an object" do
-        assert_equal "{&quot;&lt;a&gt;&quot;=&gt;1}", @filter.inspect("<a>" => 1)
+        assert_match %r|{&quot;&lt;a&gt;&quot; *=&gt; *1}|, @filter.inspect("<a>" => 1)
       end
 
       should "quote strings" do
Only in jekyll-4.3.4/test: test_filters.rb~
